ypkg2/rc.yml

```diff
@@ -399,6 +399,16 @@ actions:
             cp ${1}.bolt ${1}
         }
         boptim
+    # Check for broken symlinks, turn from error into warning by setting SYMLINK_WARN
+    - symlink_check: |
+        if test -n "$(find $installdir -xtype l)"; then
```

Comment:
Any particular reason this is not using `[[ -n "$(find ...)" ]]` bash built-ins like it does below?

Reply:
I think that was just a result of this growing organically while testing. ^^
Now using `[[ ]]` here too as it works the same.